test/integration/backend_version_test.rb

```diff
@@ -118,7 +116,7 @@ def setup
 
     Service.save! id: @service.id, provider_key: @provider_key, backend_version: 'oauth'
 
-    get '/transactions/authorize.xml', :provider_key => @provider_key,
+    get '/transactions/oauth_authorize.xml', :provider_key => @provider_key,
```

Comment:
Why are all of these calls changed to `oauth_authorize` ?

Reply:
I think that's the correct endpoint because just above the backend version changed to `oauth`

```ruby
backend_version: 'oauth'
```